x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/beta_badge_props.tsx

```diff
@@ -8,16 +8,6 @@
 import { i18n } from '@kbn/i18n';
 
 export const betaBadgeProps = {
-  label: i18n.translate('xpack.triggersActionsUI.betaBadgeLabel', {
-    defaultMessage: 'Beta',
-  }),
-  tooltipContent: i18n.translate('xpack.triggersActionsUI.betaBadgeDescription', {
-    defaultMessage:
-      'This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.',
-  }),
-};
-
-export const technicalPreviewBadgeProps = {
```

Comment:
We should rename this to `techincalPreviewBadgeProps` to avoid future confusion.